iroh/src/node/builder.rs

```diff
@@ -458,8 +458,8 @@ where
                         Ok((msg, chan)) => {
                             handler.handle_rpc_request(msg, chan);
                         }
-                        Err(_) => {
-                            info!("last controller dropped, shutting down");
+                        Err(e) => {
+                            info!("internal rpc request error: {:?}", e);
                             break;
```

Comment:
you are still breaking, how does this fix things?

Reply:
oops, i wasn't intending to. got this back somehow while switching branches.